docs/tutorials/runtime-module-in-own-crate.md

```diff
@@ -0,0 +1,167 @@
+If you plan for third parties to use your module, you should develop against the `v1.0` branch as the template demonstrates. It is also acceptable to develop against specific git commit revisions.
+
+```toml
+# This dependency is published on crates.io so just use a version
+[dependencies.parity-codec]
+default-features = false
+features = ['derive']
+version = '3.5'
+
+# These dependencies specify a git branch
+[dependencies.support]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'srml-support'
+branch = 'v1.0'
+
+[dependencies.system]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'srml-system'
+branch = 'v1.0'
+# Could also use a specific revision
+# rev = '<commit hash>'
+```
+
+The final section of the `Cargo.toml` file specifies the dev dependencies, and is not necessary to make the template compile. Remove that final section to confirm this for yourself.
+
+At this point we have a runtime module that compiles. It comes with a test, so let's see if that test passes.
+```bash
+cargo test
+```
+
+Turns out we'll need some additional dependencies to make the test pass. You can see this from the error messages, or by looking at the test module itself. Let's add `primitives`, `runtime-primitives`, and `runtime-io` as dev dependencies. The dev dependencies do not need to be listed as part of the `std` feature because they will not be part of the release.
+
+Add these lines back to your `Cargo.toml`.
+```toml
+
+[dev-dependencies.primitives]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'substrate-primitives'
+branch = 'v1.0'
+
+[dev-dependencies.runtime-primitives]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'sr-primitives'
+branch = 'v1.0'
+
+[dev-dependencies.runtime-io]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'sr-io'
+branch = 'v1.0'
+```
+
+It will take a little experimentation to gain familiarity with what dependencies you need for each module you dream up. So the take-away from this section is not that `Cargo.toml` should always look like it does in the template. Rather, `Cargo.toml` needs to have the correct dependencies and you now know how to specify them.
+
+## Add your Runtime Module to your Node
+With our runtime module now compiling and passing tests (well, passing test anyway) we're ready to add it to our node template.
+
+First we need to add our newly-created crate as a dependency in the node runtime's `Cargo.toml`.
+
+```toml
+[dependencies.test_module]
+default_features = false
+path = "../../test_module"
+# Optionally specify a version
```

Comment:
When would they want to do this?

Reply:
Removed. I guess it only makes sense if we had taught them to publish the module somewhere.